cxx-qt/src/lib.rs

```diff
@@ -10,8 +10,11 @@ use cxx_qt_gen::{extract_qobject, generate_qobject_rs};
 
 /// Read the C++ namespace prefix that cxx-qt-build has set for us
 fn read_cpp_namespace_prefix() -> Vec<String> {
-    let dir_target = std::env::var("CARGO_MANIFEST_DIR").expect("Could not get manifest dir");
-    let path = format!("{}/target/cxx-qt-gen/cpp_namespace_prefix.txt", dir_target);
+    let dir_target = std::env::var("OUT_DIR")
+        .map(|s| s + "/../../../..")
```

Comment:
This code seems a bit magical here, what is it trying to do ?

I would have thought it should be looking inside `OUT_DIR/cxx-qt-gen/cpp_namespace_prefix.txt` ? Or are there problems going across the crates.

As one of our next steps for the CMake is to instead use a CLI tool to do the generation so that we don't have this mess with the paths and can avoid cargo build happening in the CMake configure stage. (see some things in #/106 but again we'll write up a github discussion shortly).